tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -392,3 +394,77 @@ def test_harvesting_arxiv_workflow_accepted(
         obj = workflow_object_class.get(obj_id)
         # It was accepted
         assert obj.status == ObjectStatus.COMPLETED
+
+        # Check the conversion for the legacy sys
+        marc_json = hep2marc.do(obj.data)
+        marcxml = legacy_export_as_marc(marc_json)
+        record = create_record(marcxml)
```

Comment:
Yeah, actually we can do even better that this: we store an XML fixture of the record how it is on INSPIRE, and we do a similar test (without checking at all the intermediate steps). Then the test compares the dictionary with MARC keys coming from this fixture and the dictionary with MARC keys coming through the workflow: in this way you don't have to type manually all these assertions, but can write just one.